src/mam/mod_mam_cache_user.erl

```diff
@@ -13,78 +11,58 @@
 %%%-------------------------------------------------------------------
 -module(mod_mam_cache_user).
 
--behaviour(gen_mod).
 -behaviour(mongoose_module_metrics).
 
 %% gen_mod handlers
--export([start/2, stop/1, supported_features/0]).
+-export([start/2]).
+-export([deps/2]).
+-export([stop/1]).
+-export([supported_features/0]).
```

Comment:
Personally, I don't like this repetitive style. It's a bit like the comma at the beginning of a line - "it's a man doing a machine's job", to quote one of our colleagues at ESL - one line more or less in a diff is not worth the extra typing and clutter IMO (and actually it's four lines instead of one right here).